tests/acceptance/features/webUITextEditor/editTextFiles.feature

```diff
@@ -22,3 +22,18 @@ Feature: textFiles
     And the user closes the text editor
     And the user opens file "New text file.txt" using the webUI
     Then line 1 of the text should be "other text before stuff"
+
+  @issue-36233
+  Scenario: Delete and restore hidden text file
+    Given the user has created a text file with the name "abc.txt"
+    And the user has input "This is a hidden file" in the text area
+    And the user has closed the text editor
+    When the user enables the setting to view hidden files on the webUI
+    And the user renames file "abc.txt" to ".abc.txt" using the webUI
```

Comment:
I think this all can be done with one single step 
```
Given user has uploaded file with content "..." to "..."
```

Reply:
We are writing test scenarios to demonstrate this problem owncloud/core#36233 so I think it would be better this way. What do think @dpakach ?

Reply:
according to the issue the problem is with restoring the hidden file so for creating hidden file I still think we can do it in single step.